src/makeInfinityAdder.js

```diff
@@ -6,7 +6,21 @@
  */
 
 function makeInfinityAdder() {
-  // write code here
+  let result = 0;
+
+  return function sum(a) {
+    if (arguments.length === 0) {
```

Comment:
it would be better to check `a` variable directly

Reply:
ok, tnx👍